app/valid.py

```diff
@@ -0,0 +1,27 @@
+from flask import abort, make_response
+import requests
+import os
+
+
+def validate_id(model, id):
+    if not id.isnumeric():
+        abort(make_response({'Error': f'{id} is invalid'}, 400))
+    
+    entity = model.query.get(id)
+    if not entity:
+        abort(make_response({'Not found': f'No {model.__name__} with id#{id} is found'}, 404))
+    return entity
+
+def validate_entry(model, request_body):
+    for atr in model.get_attributes():
```

Comment:
Nicely done! This wasn't required, but I like how you're validating and using `get_attributes()`